lib/audit/catalog_to_moab.rb

```diff
@@ -139,4 +139,24 @@ def online_moab_found?(druid, storage_dir)
     false
   end
 
+  # given whether the caller found the expected version of preserved_copy on disk, this will perform
+  # other validations of what's on disk, and will update the status accordingly.
+  # TODO: near duplicate of method in POHandler - extract superclass or moab wrapper class??
```

Comment:
dang it!  OC me also puts a similar message in POHandler by the corresponding method ...